spec/support/shared/examples/msf/db_manager/session.rb

```diff
@@ -39,12 +39,15 @@
           let(:module_instance) do
             name = 'multi/handler'
 
-            double(
-                'Msf::Module',
+            d = double(
+                'Msf::Exploit',
+                :user_data => user_data,
```

Comment:
Since you're changing everything else in this let, I would just go ahead and change this to 1.9 JSON hash syntax.